### ISSUE TITLE
Can't run "quasar build" in the root of a docker image

### ISSUE THREAD

**Issue by mgkeen** [quasarframework/quasar#6871]
**Describe the bug**
I've been writing a dockerfile to produce an image of our app for production. 

If as part of the build I copy the project files to the root of the image, then i get the failure `app:paths ⚠️  Error. This command must be executed inside a Quasar v1+ project folder.` when trying to run `quasar build`.

If I instead create a directory `/app` in the container and copy the files there then `quasar build` runs succesfully.

Failing dockerfile:
```
FROM node as build

ENV YARN_VERSION 1.21.1

RUN yarn policies set-version ${YARN_VERSION}
RUN yarn global add @quasar/cli

COPY package.json ./
COPY yarn.lock ./
RUN yarn

COPY . .
RUN yarn run build
```

Working dockerfile:
```
FROM node as build

ENV YARN_VERSION 1.21.1

RUN yarn policies set-version ${YARN_VERSION}
RUN yarn global add @quasar/cli

COPY package.json ./
COPY yarn.lock ./
RUN yarn

COPY . ./app
WORKDIR /app
RUN yarn run build
```

This is in a fresh project created by the quasar cli.

I think this is due to the check in this line, and more generally the app directory not expecting to be at the root of the file system:

`https://github.com/quasarframework/quasar/blob/6d2c3ffdbab995532e078754604bc5ad13dd11bd/app/lib/app-paths.js#L10`

**Comment by smolinari** [comment]
The difference between the two is, in your working dockerfile, you have `WORKDIR`.  That tells docker to run the command in that directory. I believe when you don't set the `WORKDIR`, docker  defaults to your user's home directory. 

Scott

**Comment by mgkeen** [comment]
The `WORKDIR` definitely isn't required in the first one. I had debugged it with ls commands to make sure I was definitely in the right directory, and even if i set the `WORKDIR` now it still fails with the same issue.

I agree the separate directory is probably better practice anyway. Is there a good place to document this or can the error message be improved? Happy to do either myself. The current error message isn't particularly useful and it took me some time to figure it out, I raised the issue so others could avoid the same pain more than anything.
